T/TDLib/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TDLib"
+version = v"1.6.10"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/tdlib/td.git", "78e45da6b9014f8efd96cee0ad5701464d911cc7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add gperf
+cd td/
+install_license LICENSE_1_0.txt
+sed -i 's|TD_HAS_MMSG 1|TD_HAS_MMSG 0|' tdutils/td/utils/port/config.h  # otherwise not compatible with old glibc
+mkdir build
+cd build
+cmake -DCMAKE_BUILD_TYPE=Release -DZLIB_LIBRARY=$libdir -DZLIB_INCLUDE_DIR=$includedir ..
+for f in $(grep -l $(find . -name link.txt) -e 'opt/bin/c++')
+do
+    # without this flag it cannot find libz
+    sed -i ' 1 s/$/ -lz/' $f
+done
+cmake --build . -j${nproc}
+cp libtdjson.so $libdir
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(p -> Sys.islinux(p) && (libc(p) == "glibc" && arch(p) == "i686" || arch(p) == "x86_64"), supported_platforms())
```

Comment:
This is simpler
```suggestion
platforms = filter(p -> Sys.islinux(p) && (libc(p) == "glibc" && proc_family(p) == "intel"), supported_platforms())
```

Reply:
it's "x64 with any libc" or "i686 with glibc"

Reply:
I know, I was just pointing out that using `proc_family(p)` gives the same condition as yours, but fewer keystrokes

Reply:
Cannot reproduce it locally, says `ERROR: UndefVarError: proc_family not defined`. But looks like your variant won't select x64 musl.

Reply:
Oh, I see, I misread your condition, I thought you were skipping that platform on purpose